driver.go

```diff
@@ -254,8 +252,8 @@ func (r *statsRows) Close() error {
 }
 func (r *statsRows) Next(dest []driver.Value) error {
 	err := r.wrapped.Next(dest)
-	if err == nil {
-		r.d.RowIterated()
+	if err != io.EOF {
```

Comment:
From what I can tell that is something that is handled by the `database/sql` package if you try and Scan but there are no rows left. So its not something that we can deal with on the driver level.